model/versioning/Versioned.php

```diff
@@ -1556,16 +1647,7 @@ public static function set_reading_mode($mode) {
 	 * @return string
 	 */
 	public static function get_reading_mode() {
-		return Versioned::$reading_mode;
-	}
-
-	/**
-	 * Get the name of the 'live' stage.
-	 *
-	 * @return string
-	 */
-	public static function get_live_stage() {
```

Comment:
Add it to the upgrade notes - I think we'll probably need to make a 3.x version that does have all the deprecation notices nearer the time